README.adoc

```diff
@@ -349,19 +510,71 @@ system-deployment-6bd97d9bf6-x4zth      1/1       Running   0          25s
 inventory-deployment-645767664f-nbtd9   1/1       Running   0          1m
 ----
 
-Wait for your two new pods to be in the ready state, then `curl -I` or visit the `{system-api}` URL. You'll notice that the `X-Pod-Name` header will have a different value when you call it multiple times. This is because there are now three pods running all serving the `system` application. Similarly, to descale your deployments you can use the same scale command with fewer replicas.
+// Static guide instruction
+ifndef::cloud-hosted[]
+Wait for your two new pods to be in the ready state, then make a `curl -I` request to, or visit the `{system-api}` URL. 
+endif::[]
+
+// Cloud-hosted guide instruction
+ifdef::cloud-hosted[]
+Wait for your two new pods to be in the ready state, then make the following `curl` command:
+
+```
+curl -I http://$SYSTEM_HOST:31000/system/properties
+```
+{: codeblock}
+endif::[]
+
+You'll notice that the `X-Pod-Name` header will have a different value when you call it multiple times. 
+This is because there are now three pods running all serving the `system` application. 
+Similarly, to descale your deployments you can use the same scale command with fewer replicas.
+
+[role='command']
+```
+kubectl scale deployment/system-deployment --replicas=1
+```
 
 == Redeploy microservices
 
-When you're building your application, you may find that you want to quickly test a change. To do that, you can rebuild your Docker images then delete and re-create your {kube} resources. Note that there will only be one `system` pod after you redeploy since you're deleting all of the existing pods.
+When you're building your application, you may find that you want to quickly test a change. 
```

Comment:
you may find that you want to quickly test a change. 
--->
you might want to quickly test a change.